src/components/KInput/KInput.vue

```diff
@@ -35,6 +35,7 @@
       <!-- TODO: [beta] change input class to text-input -->
       <input
         v-bind="modifiedAttrs"
+        :aria-describedby="helpTextId"
```

Comment:
```suggestion
        :aria-describedby="helpText ? helpTextId : undefined"
```

issue: if `helpText` is empty it won't be rendered so we shouldn't reference non-existent element in `aria-describedby` attribute here